### PR TITLE
[DG] Log when we get an error from Twilio

### PR DESCRIPTION
**Pull request by danielglauser**
We were swallowing errors from Twilio. Let's not do that anymore.

### CODE REVIEW

File: src/sms_works/core.clj [hunk 1]
```diff
@@ -17,20 +17,29 @@
...
-                        :To to
-                        :Body body}))))
+        from (or from (select-number))
+        twilio-resp (twilio/with-auth sid token
+                      (twilio/send-sms {:From from
+                                        :To to
+                                        :Body body}))]
+    twilio-resp))
+
+(defn handle-twilio-resp
```
Comment (cap10morgan) [inline src/sms_works/core.clj]: This seems more like a `log-twilio-resp` than a `handle-twilio-resp`. "handle" made me expect it to do something w/ business logic w/ the response.

**Comment by danielglauser** [comment]
@cap10morgan Thoughts kind sir?

File: src/sms_works/core.clj [hunk 2]
```diff
@@ -17,20 +17,29 @@
...
 
 (defn message-handler
   [ch meta ^bytes payload]
   (let [message (String. payload "UTF-8")]
     (log/info "Handling message")
-    (log/debug message)
-    (-> message
-        edn/read-string
-        send-sms)))
+    (log/info message)
```
Comment (cap10morgan) [inline src/sms_works/core.clj]: This should remain at the `debug` level.

**Comment by cap10morgan** [comment]
One small little change to make, fix merge conflicts, and this will be LGTM.

**Comment by danielglauser** [comment]
Looks like this is no longer needed, the functionality is already in master. @cap10morgan Unless you object I'm going to delete this.

**Comment by cap10morgan** [comment]
@danielglauser sounds good

**Pull request closed without merging**